db/schema.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS tenants;
+DROP TABLE IF EXISTS apartments;
+
+CREATE TABLE tenants (
+  id SERIAL PRIMARY KEY,
+  name TEXT NOT NULL,
+  age INT,
+  gender TEXT,
+  apartment_id INT
+);
+CREATE TABLE apartments (
+  id SERIAL PRIMARY KEY,
+  address TEXT,
+  monthly_rent INT,
+  sqft INT,
+  num_beds INT,
+  num_baths INT
+);
```

Comment:
i'd all the apartment attributes NOT NULL-- maybe not address or bed and bath if you have a really sketchy landlord though :shipit: